boards/BMS/Src/Io/Io_CurrentSense.c

```diff
@@ -68,31 +80,31 @@ ExitCode Io_CurrentSense_ConvertToHighResolutionMainCurrent(float adc_voltage, f
     //
     // Sensitivity = 6.67mV/A
 
-    const float offset             = 2.5f;
-    const float sensitivity        = 6.67e-3f;
-    const float hsnbv_d06_output_2 = adc_voltage * (1e3f + 2.2e3f) / 2.2e3f;
+    const float hsnbv_d06_output_2 = adc_voltage * 1 / OUTPUT_2_DIV;
```

Comment:
could be nice to clarify with a small comment that this is the voltage before the divider. Up 2 u tho im not super picky